Update mid-function-visioshapesheet.md (#696)
MicrosoftDocs/office-developer-client-docs: docs/visio/mid-function-visioshapesheet.md

- Less than 1, MID returns the #VALUE! error value.

> If  *num_chars*  is negative, MID returns the characters up to the end of *text*.

Reviewer: ```suggestion
If  *num_chars*  is negative, MID returns the characters in *text* string.
```
